pricegraph/Cargo.toml

```diff
@@ -11,7 +11,7 @@ harness = false
 [dependencies]
 arbitrary = { version = "0.4", optional = true, features = ["derive"] }
 petgraph = "0.5"
-primitive-types = "0.7"
+primitive-types = "0.6.2"
```

Comment:
I find this weird because it looks like you removing the imports of this. I  guess then we are still using it somewhere.

Reply:
It's transitively imported by our web3 dependency. There we import an older version that the current latest. By making our explicit imports use the same version as our indirect dependency inclusion, we don't have to explicitly convert types that are basically the same (but not exactly since different version)